src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -12,7 +12,15 @@
 
     {% for crumb in crumbs %}
       {% if forloop.last == true %}
-        <li><a onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" aria-current="page" href="/{{ path }}">{{ crumb.text }}</a></li>
+        <li>
+            <a 
+              class="breadcrumb-link" 
+              onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" 
+              aria-current="page" 
```

Comment:
Leaving this link here as a breadcrumb: https://www.w3.org/TR/wai-aria/#aria-current